### PR TITLE
add markdown viewer

### PR DESCRIPTION
**Pull request by kevodwyer**
Also see Peergos PR: https://github.com/Peergos/Peergos/pull/912

### CODE REVIEW

File: src/views/Drive.vue [hunk 1]
```diff
@@ -783,25 +822,33 @@ module.exports = {
...
-		    else if (app == "identity-proof")
-			this.showIdentityProof = true;
-		    else if (app == "hex")
-			this.showHexViewer = true;
-		    else if (app == "search")
-			this.showSearch = true;
-
-                    this.updateHistory(app, this.getPath, "");
+            let that = this;
+            Vue.nextTick(() => {
```
Comment (ianopolous) [inline src/views/Drive.vue:836]: Can you explain the need for nextTick?

File: src/components/App.vue [hunk 2]
```diff
@@ -77,6 +77,7 @@ const Social = require("../views/Social.vue");
 const Calendar = require("../views/Calendar.vue");
 const Chat = require("../views/Chat.vue");
 const Email = require("../views/Email.vue");
+const Markdown = require("../components/viewers/markdown/Markdown.vue");
```
Comment (ianopolous) [inline src/components/App.vue:80]: Is this still needed?

File: src/views/Drive.vue (hunk 1 above)
Comment (kevodwyer) [reply src/views/Drive.vue:850]: I have added comments. Required due to side-effect of history navigation

File: src/components/App.vue (hunk 2 above)
Comment (kevodwyer) [reply src/components/App.vue]: unused code removed

**Pull request merged**